README.md

```diff
@@ -147,7 +147,7 @@ Downloading Terasology via the launcher can fail for multiple reasons including:
 To check whether GitHub is available, please refer to https://www.githubstatus.com/.
 
 To check whether the Terasology artifact is available, please go to our [Terasology release page](https://github.com/MovingBlocks/Terasology/releases) and look for the version you were trying to download.
-You should find it at https://github.com/MovingBlocks/Terasology/releases/tag/v<version>.
+You should find it at `https://github.com/MovingBlocks/Terasology/releases/tag/v<version>`, e.g. `https://github.com/MovingBlocks/Terasology/releases/tag/v5.3.0`.
```

Comment:
I've put this in backticks because of the angular braces (which are otherwise interpreted as HTML tags).

The concrete example does not have them, so we can make that a clickable link by omitting the backticks there:

```suggestion
You should find it at `https://github.com/MovingBlocks/Terasology/releases/tag/v<version>`, e.g. https://github.com/MovingBlocks/Terasology/releases/tag/v5.3.0.
```